vars/test.yml

```diff
@@ -389,7 +389,7 @@ windows_nfs_msu_path: ''
 # This paramter can also be used in below test case "windows_online_update".
```

Comment:
The test case in this line also needs to update.

Reply:
Yeah, thanks.